triple-store/triple-store-impl-blazegraph/src/main/java/com/powsybl/triplestore/impl/blazegraph/TripleStoreBlazegraph.java

```diff
@@ -378,17 +397,62 @@ private static void copyNamespacesToModel(RepositoryConnection conn, Model m) th
     }
 
     private static void addNamespaceForBase(RepositoryConnection cnx, String base) throws RepositoryException {
-        cnx.setNamespace("data", base + "#");
+        cnx.setNamespace("data", base + "/#");
     }
 
     private Resource context(RepositoryConnection conn, String contextName) {
         String name1 = contextName.replace(namespaceForContexts(), "");
         return conn.getValueFactory().createURI(namespaceForContexts(), name1);
     }
 
+    @Override
+    public void addNamespace(String prefix, String namespace) {
+        RepositoryConnection cnx = null;
+        try {
+            cnx = repo.getConnection();
+            cnx.setNamespace(prefix, namespace);
+        } catch (RepositoryException x) {
+            throw new TripleStoreException(String.format("Adding namespace %s", namespace), x);
+        } finally {
+            if (cnx != null) {
+                try {
+                    cnx.close();
+                } catch (RepositoryException x) {
+                    LOG.error(x.getMessage());
+                }
+            }
```

Comment:
Maybe you could use the new `closeConnection` method here?

Reply:
done